torch/_dynamo/symbolic_convert.py

```diff
@@ -1951,7 +1955,9 @@ class InliningInstructionTranslator(InstructionTranslatorBase):
 
     @classmethod
     def inline_call(cls, parent, func, args, kwargs):
-        with patch.dict(counters, {"unimplemented": counters["inline_call"]}):
+        with patch.dict(
+            counters, {"unimplemented": counters["inline_call"]}
+        ), TracingContext.current_frame(parent.frame_summary()):
```

Comment:
Isn't the more logical place to patch this `inline_call_`?

Reply:
(assuming we're doing caller patches, not callee patches)

Reply:
Yes, that is a tad closer, although to me it's a detail, since its still within the implementation secrets of the instruction translator.

Reply:
Actually, this might be spurious...